src/components/application_manager/rpc_plugins/rc_rpc_plugin/include/rc_rpc_plugin/resource_allocation_manager.h

```diff
@@ -55,6 +55,13 @@ namespace ResourceState {
 enum eType { FREE = 0, BUSY };
 }
 
+/**
+ * Defines triggers for OnRCStatus notification sending
+ */
+namespace NotificationTrigger {
+enum eType { APP_REGISTRATION = 0, MODULE_ALLOCATION, STATE_CHANGING };
```

Comment:
@LitvinenkoIra describe triggers in doxygen documentation, and rename STATE_CHANGING to RC_STATE_CHANGING